docs/source/notes/randomness.rst

```diff
@@ -2,70 +2,73 @@ Reproducibility
 ===============
 
 Completely reproducible results are not guaranteed across PyTorch releases,
-individual commits or different platforms. Furthermore, results need not be
+individual commits, or different platforms. Furthermore, results need not be
 reproducible between CPU and GPU executions, even when using identical seeds.
 
-However, in order to make computations deterministic on your specific problem on
-one specific platform and PyTorch release, there are a couple of steps to take.
+However, there are some steps you can take to limit the number of sources of
+nondeterministic behavior for a specific platform, device, and PyTorch release.
 
-There are two pseudorandom number generators involved in PyTorch, which you will
-need to seed manually to make runs reproducible. Furthermore, you should ensure
-that all other libraries your code relies on and which use random numbers also
-use a fixed seed.
+PyTorch uses a pseudorandom number generator, which you will need to seed
+manually to make it act deterministically. Furthermore, you can configure
+PyTorch to avoid using nondeterministic algorithms for a subset of operations.
+Finally, if your code relies on other libraries that use random numbers, make
+sure to give them a fixed seed.
 
-PyTorch
-.......
+Random number generation
+........................
 You can use :meth:`torch.manual_seed()` to seed the RNG for all devices (both
 CPU and CUDA)::
 
     import torch
     torch.manual_seed(0)
 
+Deterministic algorithms
+........................
+Some PyTorch operations use nondeterministic algorithms by default. A subset
+of them can be configured to use a deterministic implementation or throw an
+error if a deterministic implementation is not available. Please check the
```

Comment:
Suggestion: 

"torch.set_deterministic" lets you configure PyTorch to use deterministic algorithms instead of nondeterministic ones where available, and to throw an error if an operation is known to be nondeterministic (and without a deterministic alternative).